### ISSUE TITLE
Provide a template for issues

### ISSUE THREAD

**Issue by parmentelat** [damianavila/RISE#489]
When creating an issue on github, users should see some recommendations, like

* providing version info 
* checking the js console (and a word on how to do that)
* trying on another slideshow
* check for custom styles

And possibly others that escape me right now

**Comment by damianavila** [comment]
Yep, let's do this. I am tired to always ask the same questions :wink:

I would probably add:

* provide the ipynb file or a link to the repo containing the ipynb file.

Let's start with your list above + the one I mentioned and we can add more things later.

**Comment by parmentelat** [comment]
gave that a shot, but as per 
https://help.github.com/en/articles/creating-issue-templates-for-your-repository
I would need access to the `Settings` tab of the main github repo, but I don't seem to have that access

**Comment by damianavila** [comment]
It seems you cannot give admin access to collaborators in personal repositories.
If you already wrote the template, please share it here and I will make the rest.
